README.md

```diff
@@ -1,4 +1,12 @@
-# MuseScore: Music notation and composition software
+<p align="center">
+   <img src="mscore/data/musescore_logo_full.png"></img><br/>
+   Music notation and composition software
+</p>
+
+<p align="center">
+	<a href="https://travis-ci.org/musescore/MuseScore"><img src="https://secure.travis-ci.org/musescore/MuseScore.svg" alt="Travis CL"></a>
```

Comment:
Just keep the markdown syntax here,: `[![Build Status](https://secure.travis-ci.org/musescore/MuseScore.svg)](https://travis-ci.org/musescore/MuseScore)`